astroquery/linelists/cdms/core.py

```diff
@@ -421,25 +429,24 @@ def get_molecule(self, molecule_id, *, cache=True, return_response=False):
                                  timeout=self.TIMEOUT, cache=cache)
         if return_response:
             return response
-        result = self._parse_cat(response)
+        result = self._parse_cat(response.text)
```

Comment:
shall we raise_for_status prior to this? Or maybe can we even upstream that raise for status and do it inside `_request()`? -- that case should/would be a follow-up rather than in this PR.